src/server/pfs/cmds/cmds.go

```diff
@@ -453,21 +459,38 @@ $ {{alias}} foo@master --from XXX`,
 	shell.RegisterCompletionFunc(listCommit, shell.RepoCompletion)
 	commands = append(commands, cmdutil.CreateAlias(listCommit, "list commit"))
 
-	var repos cmdutil.RepeatedStringArg
-	flushCommit := &cobra.Command{
-		Use:   "{{alias}} <repo>@<branch-or-commit> ...",
-		Short: "Wait for all commits caused by the specified commits to finish and return them.",
-		Long:  "Wait for all commits caused by the specified commits to finish and return them.",
+	var branches cmdutil.RepeatedStringArg
+	waitCommit := &cobra.Command{
```

Comment:
The naming should be consistent between pachctl and the Go client. I kind of prefer wait because it seems a bit clearer, but I'm not strongly against block.

Reply:
Also, it doesn't seem like a good idea to merge the commitset wait and commit wait into the same command. This code would be simpler if they were separate and I feel like it would be better to have a more explicit API.

Reply:
Yeah, I'm planning to rename all the `Block.*` stuff to `Wait.*` after this (assuming everyone is on board) - definitely got some comments that 'block' isn't very clear about what it means.